lib/ipmi_delloem.c

```diff
@@ -1338,7 +1338,7 @@ ipmi_lcd_set_single_line_text(struct ipmi_intf * intf, char * text)
  *                  0 if successful
  */
 static int
-ipmi_lcd_set_text(struct ipmi_intf * intf, char * text, int line_number)
+ipmi_lcd_set_text(struct ipmi_intf * intf, char * text, int line_number __attribute__((unused)))
```

Comment:
Right, this builds on other systems -- Will fix.  Also, will examine each individual method to determine whether the parameters are for consistency of calling, or due to a function pointer signature.

Reply:
Ok, so that doesn't exist in helper.h, so you're asking me to add it, and then use it.  I'm going to stage a couple patches in this PR:
1) add macro
2) fix all the places it wasn't already used (plenty of __attribute__((unused)) in the codebase)
3) then fix my patchset to use it.